doc/collectorPipeline.md

```diff
@@ -280,81 +369,63 @@ stateDiagram-v2
-  state discoveryCollectorDeployment {
-
-    dc_metricsPipeline: 'metrics' pipeline
-    state dc_metricsPipeline {
-      dc_r0: 'forward/metric-exporter' connector
-      dc_e0: 'otlp' exporter
-      dc_r0 --> dc_e0 : processors
-    }
-
-    dc_metricsDiscoveryScrapePipeline: 'metrics/discovery-scrape' pipeline
-    state dc_metricsDiscoveryScrapePipeline {
-      dc_r1: 'receiver_creator/discovery' receiver
-      dc_e1: 'routing/discovered_metrics' connector
-      dc_r1 --> dc_e1 : processors
-    }
-
-    dc_metricsDiscoveryIstioPipeline: 'metrics/discovery-istio' pipeline
-    state dc_metricsDiscoveryIstioPipeline {
-      dc_r2: 'routing/discovered_metrics' connector
-      dc_e2a: 'forward/metric-exporter' connector
-      dc_e2b: 'forward/relationship-state-events-workload-workload' connector
-      dc_e2c: 'forward/relationship-state-events-workload-service' connector
-      dc_r2 --> dc_e2a : processors
-      dc_r2 --> dc_e2b : processors
-      dc_r2 --> dc_e2c : processors
-    }
-
-    dc_metricsRelationshipWorkloadPipeline: 'metrics/relationship-state-events-workload-workload-preparation' pipeline
-    state dc_metricsRelationshipWorkloadPipeline {
-      dc_r3: 'forward/relationship-state-events-workload-workload' connector
-      dc_e3: 'solarwindsentity/istio-workload-workload' connector
-      dc_r3 --> dc_e3 : processors
-    }
-
-    dc_metricsRelationshipServicePipeline: 'metrics/relationship-state-events-workload-service-preparation' pipeline
-    state dc_metricsRelationshipServicePipeline {
-      dc_r4: 'forward/relationship-state-events-workload-service' connector
-      dc_e4: 'solarwindsentity/istio-workload-service' connector
-      dc_r4 --> dc_e4 : processors
-    }
-
-    dc_logsStateEventsPipeline: 'logs/stateevents' pipeline
-    state dc_logsStateEventsPipeline {
-      dc_r5a: 'solarwindsentity/istio-workload-workload' connector
-      dc_r5b: 'solarwindsentity/istio-workload-service' connector
-      dc_e5: 'otlp' exporter
-      dc_r5a --> dc_e5 : processors
-      dc_r5b --> dc_e5 : processors
-    }
-
-    dc_metricsDiscoveryCustomPipeline: 'metrics/discovery-custom' pipeline
-    state dc_metricsDiscoveryCustomPipeline {
-      dc_r6: 'routing/discovered_metrics' connector
-      dc_e6: 'forward/metric-exporter' connector
-      dc_r6 --> dc_e6 : processors
-    }
-
-    dc_metricsDiscoveryScrapePipeline --> dc_metricsDiscoveryCustomPipeline
-    dc_metricsDiscoveryScrapePipeline --> dc_metricsDiscoveryIstioPipeline
-    dc_metricsDiscoveryIstioPipeline --> dc_metricsRelationshipWorkloadPipeline
-    dc_metricsDiscoveryIstioPipeline --> dc_metricsRelationshipServicePipeline
-    dc_metricsRelationshipWorkloadPipeline --> dc_logsStateEventsPipeline
-    dc_metricsRelationshipServicePipeline --> dc_logsStateEventsPipeline
-
-    dc_metricsDiscoveryCustomPipeline --> dc_metricsPipeline
+    nc_metricsDiscoveryIstioPipeline --> nc_metricsPipeline
   }
 ```
+
+## Pipeline Overview
```

Comment:
Please remove this whole block, it is very verbose description that noone will read. Keep just mermaid